src/components/atoms/button.test.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+
+import Button from './button';
+
+describe('Button component', () => {
+	it('snapshot button component', () => {

```

Comment:
its better to have complete sentences to read all whats is happening in the console
like it("should do something")